ES/ESmy020.xml

```diff
@@ -36,305 +36,277 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
 
                   <msItem xml:id="ms_i1">
-                    <locus from="1r" to="74rb"/>
-                    <title type="complete" ref="LIT1931Mashaf"/>
-
-                    <msItem xml:id="ms_il.1">
-                    <locus from="2ra" to="2vb"/>
-                    <title type="incomplete">Mǝlṭān- and ʾaryām-antiphons for priests and deacons</title>
-                    <incipit>
-                    <q xml:lang="gez">
-                      </q></incipit>
-                      <explicit>
-                      <q xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ
-                      </q></explicit>
-                    <note>The text is the end of <ref corresp="#ms_il.6"/>; the leaf is placed in the reversed position,
-                    to be read from <foreign xml:lang="gez">ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</foreign>, <locus target="2va"/>)</note>
+                  <locus from="2ra" to="71rb"/>
+                  <title type="incomplete" ref="LIT1931Mashaf"/>
                   </msItem>
```

Comment:
msItem ms_i1 must be closed after msItem xml:id="ms_i1.21" and before msItem ms_i2, not here, or the nesting is all wrong